generators/server/templates/build.gradle.ejs

```diff
@@ -287,6 +287,7 @@ dependencies {
         exclude group: "org.springframework.boot", module: "spring-boot-starter-aop"
         exclude group: "org.springframework.boot", module: "spring-boot-starter-web"
     }
+    implementation 'org.simplejavamail:simple-java-mail:6.4.3'
```

Comment:
Can we pull this version into the constants file?

Reply:
:heavy_check_mark: